pkg/minikube/cluster/cluster.go

```diff
@@ -740,34 +616,14 @@ func GetKubernetesEndpointsWithNamespace(namespace string) (endpointGetter, erro
 
 // EnsureMinikubeRunningOrExit checks that minikube has a status available and that
 // that the status is `Running`, otherwise it will exit
-func EnsureMinikubeRunningOrExit(api libmachine.API) {
+func EnsureMinikubeRunningOrExit(api libmachine.API, exitStatus int) {
```

Comment:
I think we can either 

1. Parameterize this function with exit code
2. Have it return a (bool, error) and handle the error however you want (in this case we'd rename it to IsMinikubeRunning)

2 seems cleaner to me.  Additionally, we can tell if there was an error accessing the machine or if the machine is simply not running.  It might be a bit of a refactor of things, but now might be a good time to do it as we need a little more out of this function.